packages/flutter/lib/src/material/text_form_field.dart

```diff
@@ -272,6 +270,10 @@ class TextFormField extends FormField<String> {
   /// initialize its [TextEditingController.text] with [initialValue].
   final TextEditingController? controller;
 
+  /// Called when the user initiates a change to the TextField's
```

Comment:
Can this be made a doc template and reused.

Reply:
Probably, I upated the PR accordingly.